frontend/integration-tests/views/crud.view.ts

```diff
@@ -167,3 +167,25 @@ export const emptyState = $('.cos-status-box').$('.text-center');
 
 export const errorMessage = $('.pf-c-alert.pf-m-inline.pf-m-danger');
 export const successMessage = $('.pf-c-alert.pf-m-inline.pf-m-success');
+
+export const clickYamlCreation = async function() {
```

Comment:
I'd put more detail in the name so there's no confusion about what it does. Maybe

```suggestion
export const clickListPageCreateYAMLButton = async function() {
```